packages/eslint-config/eslint-config.json

```diff
@@ -0,0 +1,66 @@
+{
+  "extends": [
+    "airbnb-base",
+    "plugin:prettier/recommended",
+    "plugin:jsdoc/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "env": {
+    "es6": true
+  },
+  "globals": {
```

Comment:
Should `"HandledPromise"` and/or `"E"` be on this list?

Reply:
They are not currently necessary to pass tests today and they could be rolled up.